### PR TITLE
[T2A4][T09-A3] Tang Di Feng

### PR DESCRIPTION
**Pull request by e0011840**

### CODE REVIEW

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -486,8 +486,12 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
         for (String[] person : getAllPersonsInAddressBook()) {
             final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
-            if (!Collections.disjoint(wordsInName, keywords)) {
-                matchedPersons.add(person);
+            for(String word: wordsInName) {
+            	for(String key: keywords){
+            		if(word.equalsIgnoreCase(key)) {
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:537]: indentation is off here!
Our coding guidelines require using 4 spaces for indentation. 
Tip: You can make Eclipse do it for you; check out how!

Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:537]: Good use of equalsIgnoreCase :+1:

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -486,8 +486,12 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
         for (String[] person : getAllPersonsInAddressBook()) {
             final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
-            if (!Collections.disjoint(wordsInName, keywords)) {
-                matchedPersons.add(person);
+            for(String word: wordsInName) {
+            	for(String key: keywords){
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:536]: need a space between `){`

File: src/seedu/addressbook/AddressBook.java [hunk 3]
```diff
@@ -34,6 +35,30 @@
  * This class is used to maintain a list of person data which are saved
  * in a text file.
  **/
+/**
+ * @author Difeng
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:39]: errr! these are quite useless comments. I am assuming they crept in by mistake?
ensure that the comments are used to explain something about the code. if you need to let the reviewer know the intention, you can use GitHub comments (like this one) to communicate the same.

File: src/seedu/addressbook/AddressBook.java [hunk 4]
```diff
@@ -383,11 +412,28 @@ private static String executeCommand(String userInputString) {
...
             executeExitProgramRequest();
+        case COMMAND_SORT_WORD:
+        	return executeSortInAlphabeticOrder();
         default:
             return getMessageForInvalidCommandInput(commandType, getUsageInfoForAllCommands());
         }
     }
-
+    
+    /* Sort in Alphabetical Order
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:422]: header comments must be in javadocs format using `/** ... */` (notice the 2 `*` in the beginning)

File: src/seedu/addressbook/AddressBook.java [hunk 5]
```diff
@@ -383,11 +412,28 @@ private static String executeCommand(String userInputString) {
             return getUsageInfoForAllCommands();
         case COMMAND_EXIT_WORD:
             executeExitProgramRequest();
+        case COMMAND_SORT_WORD:
+        	return executeSortInAlphabeticOrder();
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:416]: please take care of indentation

File: src/seedu/addressbook/AddressBook.java [hunk 6]
```diff
@@ -1092,7 +1138,8 @@ private static String getUsageInfoForAllCommands() {
                 + getUsageInfoForDeleteCommand() + LS
                 + getUsageInfoForClearCommand() + LS
                 + getUsageInfoForExitCommand() + LS
-                + getUsageInfoForHelpCommand();
+                + getUsageInfoForHelpCommand() + LS
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:1141]: you can rename `LS` to something more meaningful. please pay attention to such locations for improving the code.

File: src/seedu/addressbook/AddressBook.java (hunk 4 above)
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:422]: additionally, the comment style should use the active verb form **sorts** instead of **sort**

**Comment by okkhoy** [comment]
@e0011840 some comments added. please take a look, ack and close the PR.

please note, you need to update the documentation and tests to reflect the new behavior of your command.

**Comment by e0011840** [comment]
Noted! Thank You!!

**Pull request closed without merging**